src/components/UsefulInfoDetails/components/WarningBox/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const WarningBox = ({ children }) => {
```

Comment:
We can have one component for the `Box` with 2 types `{ warn | info }`, no need for two components having just some different css classes.